src/shogun/lib/CerealTest.cpp

```diff
@@ -0,0 +1,958 @@
+/*
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
```

Comment:
For later:
Use BSD license and your own name
See https://gist.github.com/karlnapf/24763634d5142f92f5213d282cf67e0f